lib/NetworkManager.js

```diff
@@ -302,6 +312,12 @@ class Request {
     this._headers = {};
     for (const key of Object.keys(payload.headers))
       this._headers[key.toLowerCase()] = payload.headers[key];
+
+    this._fromMemCache = false;
```

Comment:
nit: let's rename into `this._fromMemoryCache`

Reply:
done